tests/unit_tests/test_wave_06.py

```diff
@@ -76,7 +76,11 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
```

Comment:
Tests work better when they are as thorough and explicit as possible.  In this test it is important to check that each inventory contains the correct values:
```suggestion
    assert item_c in jesse.inventory
    assert item_d in jesse.inventory
    assert item_e in jesse.inventory
    assert item_a in tai.inventory
    assert item_b in tai.inventory
```